checkstyle.xml

```diff
@@ -70,15 +70,19 @@ page at http://checkstyle.sourceforge.net/config.html -->
   <!-- All Java AST specific tests live under TreeWalker module. -->
   <module name="TreeWalker">
 
-    <!-- required for SupressionCommentFilter and SuppressWithNearbyCommentFilter -->
-    <module name="FileContentsHolder"/>
-
     <!--
 
     IMPORT CHECKS
 
     -->
 
+    <module name="SuppressionCommentFilter">
```

Comment:
Why rearrange this? Doesn't look like it is moved out side of TreeWalker? 
From the change log of release it looks like only removing FileContentsHolder should be sufficient.

Reply:
This is actually because of release 8.1, `SuppressionCommentFilter ` is no longer child of Checker but `TreeWalker` http://checkstyle.sourceforge.net/releasenotes.html#Release_8.1

Reply:
Sounds good!